### PR TITLE
Added ematipico to the core contributors team

### PR DESCRIPTION
**Pull request by ematipico**
As per title, adding myself to the core contributors team.
Updated `GOVERNANCE.md` and `team.md`

### CODE REVIEW

**Comment by sebmck** [comment]
Yessss! The list is alphabetical so you can move yourself up. We should make the team page randomize order on load though. Should be easy with some JS magic, I'll open an issue.

File: website/src/contributing/team.md [hunk 1]
```diff
@@ -42,4 +42,8 @@ layout: layouts/page.njk
 		<img src="https://github.com/yassere.png?s=176">
 		<p><a href="https://github.com/yassere">Yasser Elassal</a></p>
 	</li>
+	<li>
+        <img src="https://github.com/ematipico.png?s=176">
+        <p><a href="https://github.com/ematipico">Emanuele Stoppa</a></p>
```
Comment (ooflorent) [inline website/src/contributing/team.md:47]: Indentation seems to be inconsistent with the previous list item.

Comment (ematipico) [reply website/src/contributing/team.md]: That's because my IDE is an idiot and it does spaces instead of tabs 😭 
Thank you, let me fix it!

**Comment by sebmck** [comment]
So excited to have you @ematipico!

**Pull request merged**